src/renderer/pkijs/generatePems/common.ts

```diff
@@ -0,0 +1,85 @@
+import { fromBER } from 'asn1js'
+import { stringToArrayBuffer, fromBase64 } from 'pvutils'
+import fs from 'fs-sync'
+import {
+  getAlgorithmParameters, getCrypto, setEngine,
+  CryptoEngine, Certificate, CertificationRequest,
+  SignedData
+} from 'pkijs'
+
+const webcrypto = new (require('node-webcrypto-ossl')).Crypto()
+setEngine('newEngine', webcrypto, new CryptoEngine({
+  name: '',
+  crypto: webcrypto,
+  subtle: webcrypto.subtle
+}))
+const crypto = getCrypto()
+
+export const generateKeyPair = async ({ signAlg, hashAlg }) => {
```

Comment:
it probably wouldn't hurt to add return type of this function, but maybe typescript infers it on it's own...

Reply:
actually this comment is for all the functions in this file

Reply:
bumping this comment: 'add return types of functions in this files'